docker-compose.yml

```diff
@@ -42,6 +33,24 @@ services:
       - "db"
       - "elasticsearch"
 
+  app-ci:
```

Comment:
I'm not convinced that this should be in the docker-compose but :man_shrugging:

Reply:
We have no other option but to use docker-compose to test our production image in the CI. And instead of having multiple docker-compose files, I prefer defining a new dedicated service.

Reply:
I agree with @sampaccoud

Reply:
I have no better options for now. I am open to suggestions.